frontend/components/Dashboard/Post/Post.tsx

```diff
@@ -442,10 +442,29 @@ const Post: React.FC<IPostProps> = ({ post, currentUser, refetch }: IPostProps)
         .post-content {
           display: grid;
           grid-column-gap: 10px;
-          grid-template-columns: 80px 1fr 80px;
+          grid-template-columns: 10px 1fr 10px;
```

Comment:
just in case I'm missing something, could we just be using padding and margin for this section/page? or are we using grid-specific things for anything?